core/src/main/java/cz/o2/proxima/repository/EntityDescriptor.java

```diff
@@ -38,6 +38,18 @@
     private final Map<String, AttributeDescriptor<?>> attributes = new HashMap<>();
 
     public Builder addAttribute(AttributeDescriptor<?> attr) {
+      String nameToCheck = attr.getName();
+      if (attr.isWildcard()) {
+        nameToCheck = attr.toAttributePrefix(false);
+      }
+      if (!attr.isProxy()
```

Comment:
I'm still not sure how that proxy magic works and if this is valid.

Reply:
Would this be solved if we allowed replacing attribute of the same type? That is, we would fail only when attributes contains the same name of different type (wildcard vs. regular).

Reply:
Ok, looks like what you proposed works. Changed.